otlp/handler.go

```diff
@@ -0,0 +1,188 @@
+package otlp
+
+import (
+	"container/list"
+	"context"
+	"fmt"
+	"hash/maphash"
+	"sync"
+	"time"
+
+	"github.com/segmentio/stats/v4"
+	colmetricpb "go.opentelemetry.io/proto/otlp/collector/metrics/v1"
+	metricpb "go.opentelemetry.io/proto/otlp/metrics/v1"
+)
+
+const (
+	// DefaultMaxMetrics is the default maximum of metrics kept in memory
+	// by the handler.
+	DefaultMaxMetrics = 5000
+
+	// DefaultFlushInterval is the default interval to flush the metrics
+	// to the OpenTelemetry destination.
+	DefaultFlushInterval = 10 * time.Second
```

Comment:
How do these interact? Will we flush faster, or start dropping messages, if we get to 5000 before ten seconds are up

Reply:
Updated the doc to explain the behavior :)